### PR TITLE
fix: fix typo in flox push

### PR DESCRIPTION
**Pull request by ghudgins**
## Proposed Changes

fixes a duplicated word in the help for `flox push`

## Release Notes

N/A

### CODE REVIEW

File: cli/flox/src/commands/push.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@ pub struct Push {
     #[bpaf(long, short, argument("path"))]
     dir: Option<PathBuf>,
 
-    /// Owner to push push environment to (default: current user)
+    /// Owner to push environment to (default: current user)
```
Comment (mkenigs) [inline cli/flox/src/commands/push.rs:36]:
```suggestion
    /// FloxHub owner to push environment to (default: current user)
```

**Review by mkenigs — APPROVED** [review]

**Review by mkenigs — APPROVED** [review]

**Review by mkenigs — APPROVED** [review]

**Review by stahnma — APPROVED** [review]

File: cli/flox/doc/flox-push.md [hunk 2]
```diff
@@ -50,10 +50,10 @@ FloxHub with local changes to the environment.
 :   Directory to push the environment from (default: current directory).
 
 `-o`, `--owner`
-:   Owner to push push environment to (default: current user).
+:   FloxHub owner to push environment to (default: current FloxHub user).
 
 `-f`, `--force`
-:   forceably overwrite the remote copy of the environment.
+:   Forceably overwrite the remote copy of the environment.
```
Comment (brendaneamon) [inline cli/flox/doc/flox-push.md:56]: **suggestion:** "Forceable" is British English. We use American English everywhere; thus, for consistency, this should be "Forcibly."

**Review by brendaneamon — COMMENTED** [review]
~Noted a minor issue with spelling consistency~

Actually, this is not right. ☝🏻 Please ignore. There is a meaningful distinction between "forceable" and "forcible," where the former is more appropriate in this case.

File: cli/flox/doc/flox-push.md (hunk 2 above)
Comment (ysndr) [reply cli/flox/doc/flox-push.md:56]: 🇺🇸

**Pull request merged**